security/pkg/pki/ca/ca.go

```diff
@@ -57,9 +57,6 @@ const (
 	// ServiceAccountNameAnnotationKey is the key to specify corresponding service account in the annotation of K8s secrets.
 	ServiceAccountNameAnnotationKey = "istio.io/service-account.name"
 
-	// The size of a private key for a self-signed Istio CA.
-	caKeySize = 2048
-
 	// The standard key size to use when generating an RSA private key
 	rsaKeySize = 2048
```

Comment:
`rsaKeySize` is only used by `GenKeyCert` to generate certificates for Istiod GRPC and webhooks. @myidpt should I lump this into this PR?